cpp/src/remote_key.cpp

```diff
@@ -84,7 +85,7 @@ std::shared_ptr<RemoteKey> createRemoteKeyFromSerialized(std::shared_ptr<Endpoin
 
 size_t RemoteKey::getSize() const { return _memorySize; }
 
-uint64_t RemoteKey::getBaseAddress() { return (uint64_t)_memoryBaseAddress; }
+uint64_t RemoteKey::getBaseAddress() { return static_cast<uint64_t>(_memoryBaseAddress); }
```

Comment:
```suggestion
uint64_t RemoteKey::getBaseAddress() { return _memoryBaseAddress; }
```

Reply:
The typecast is redundant since [the attribute already is `uint64_t`](https://github.com/rapidsai/ucxx/blob/be0965b67390cdc78fc916ddf8ea4bf8bd8259d0/cpp/include/ucxx/remote_key.h#L37), we're safe to just remove it then.